src/components/modules/blockManager.js

```diff
@@ -3,9 +3,12 @@
  * @classdesc Manage editor`s blocks storage and appearance
  *
  * @module BlockManager
+ *
+ * @version 2.0.0
  */
 
 import Block from '../block';
+import Selection from '../Selection';
```

Comment:
`Selection` is a Global window-scoped variable

Reply:
Selection used in Codex Editor instance scope, so that it won't make a trouble.